app/src/main/java/ch/epfl/sdp/item/Healthpack.java

```diff
@@ -15,6 +16,11 @@ public Healthpack(double healthPackAmount) {
         this.healthPackAmount = healthPackAmount;
     }
 
+    @Override
+    public Item clone() {
+        return new Healthpack(healthPackAmount);
+    }
+

```

Comment:
Just a small question, why we need clone method for items here? Any purpose or usage in the product code?

Reply:
Alex, the assistant, said we should do defensive copies of our objects, as a good practice.

Reply:
It is also useful as we use Item references as key for Maps